holidays/countries/syrian_arab_republic.py

```
@@ -0,0 +1,142 @@
+    SEP,
+    OCT,
+    NOV,
+    DEC,
+    FRI,
+    SAT,
+    SUN,
+)
+from holidays.constants import GOVERNMENT, OPTIONAL, PUBLIC
+from holidays.groups import InternationalHolidays, IslamicHolidays, StaticHolidays
+from holidays.holiday_base import HolidayBase
+class SYR(SyrianArabRepublic):
```

Ruff (0.8.2)

Undefined name `SyrianArabRepublic`

(F821)